packages/wrangler/src/deploy/utils.ts

```diff
@@ -0,0 +1,44 @@
+import { existsSync } from "node:fs";
```

Comment:
I'd rather this lived [here](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/experimental-assets.ts) as it is not deploy specific. We do smth similar in `[dev](https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/dev.tsx#L807-L833)` as well. Careful tho with https://github.com/cloudflare/workers-sdk/blob/main/packages/wrangler/src/dev.tsx#L832 which `dev` needs, but is not relevant to `deploy`

Reply:
Good catch – thanks for letting me know about dev.tsx!

Moved in https://github.com/cloudflare/workers-sdk/pull/6379/commits/d16520472ba5fbf2805e654308ed16cac79c5aac

Reply:
thank you Rahul!